config.js

```diff
@@ -30,6 +30,9 @@ var config = {
     // BOSH URL. FIXME: use XEP-0156 to discover it.
     bosh: '//jitsi-meet.example.com/http-bind',
 
+    // Websocket URL
+    // websocket: 'wss://jitsi-meet.example.com/xmpp-websocket',
```

Comment:
We may eventually want to have an example comment for multidomain support like so:

    bosh: '//<!--# echo var="http_host" default="jitsi-meet.example.com" -->/<!--# echo var="subdir" default="" -->http-bind',
        websocket: 'wss://<!--# echo var="http_host" default="jitsi-meet.example.com" -->/<!--# echo var="subdir" default="" -->xmpp-websocket',

Reply:
on second thought,  it is in the nginx example and config.js is more as a doc/reference, so I will skip that if you're ok with it? just would rather not pollute it too much

Reply:
We can do more in the examples directory instead

Reply:
it is kind of crazy how many examples are there...

Reply:
We were talking with Saul in the next PR to delete that example filder